examples/get-handled-paymail-domains.ts

```diff
@@ -0,0 +1,19 @@
+import { SpvWalletClient } from '../dist/typescript-npm-package.cjs.js';
+import { exampleAdminKey, exampleXPriv } from './example-keys.js';
+import { errMessage } from './utils.js';
+
+const server = 'http://localhost:3003/v1';
+
+// It is possible to get paymail domains either as an admin or as a user.
+if (!exampleAdminKey && !exampleXPriv) {
+  console.log(errMessage('adminKey'), 'or', errMessage('xPriv'));
```

Comment:
This will print out 3 strings - we should use a template string `${errMessage('adminKey')} or ${errMessage('xPriv')}`

Reply:
what's the difference, it is stil the same message?

Reply:
just a formatting issue - if it's not a problem, ignore it